tests/test_events/test_seed_id.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Feb 20 14:35:58 2019
+
+@author: shawn
+"""
+
+import pytest
+
+import obspy
+import obsplus
+
+
+# I'm not sure of the best place to put these tests, feel free to move them
```

Comment:
I think the end of obsplys/tests/tests_events/test_catalog_utils.py would be the correct place.